GooglePlayInstant/Editor/PlayInstantQuickDeployHttpHandler.cs

```diff
@@ -0,0 +1,361 @@
+using NUnit.Framework.Constraints;
+using UnityEngine;
+using Random = System.Random;
+
+[assembly: InternalsVisibleTo("GooglePlayInstant.Tests.Editor.QuickDeploy")]
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// A Class with utility methods for sending GET and POST requests
+    /// </summary>
+    public static class QuickDeployWwwRequestHandler
+    {
+        /// <summary>
+        /// Sends a general POST request to the provided endpoint, along with the data provided in the byte array and header
+        /// parameters
+        /// </summary>
+        /// <param name="endpoint">Endpoint to which the data should be sent </param>
+        /// <param name="postData">An array of bytes representing the data that will be passed in the POST request body</param>
+        /// <param name="postHeaders"> A dictionary representing a set of key-value pairs to be added to the request headers</param>
+        /// <returns></returns>
+        public static string SendHttpPostRequest(string endpoint, byte[] postData,
+            Dictionary<string, string> postHeaders)
+        {
+            var form = new WWWForm();
+            if (postHeaders != null)
+            {
+                foreach (var pair in postHeaders)
+                {
+                    form.headers.Add(pair.Key, pair.Value);
+                }
+            }
+
+            var www = new WWW(endpoint, postData, form.headers);
+            return GetResultWhenDone(www);
+        }
+
+        /// <summary>
+        /// Sends a general POST request to the provided endpoint, along with the data provided in the form and headers
+        /// parameters
+        /// </summary>
+        /// <param name="endpoint">Endpoint to which the data should be sent </param>
+        /// <param name="postForm">A dictionary representing a set of key-value paris to be added to the request body</param>
+        /// <param name="postHeaders"> A dictionary representing a set of key-value pairs to be added to the request headers</param>
+        /// <returns></returns>
+        public static string SendHttpPostRequest(string endpoint, Dictionary<string, string> postForm,
+            Dictionary<string, string> postHeaders)
+        {
+            var form = new WWWForm();
+            if (postForm != null)
+            {
+                foreach (var pair in postForm)
+                {
+                    form.AddField(pair.Key, pair.Value);
+                }
+            }
+
+            return SendHttpPostRequest(endpoint, postForm != null ? form.data : null, postHeaders);
+        }
+
+        /// <summary>
+        /// Sends a general GET request to the specified endpoint along with specified parameters and headers
+        /// </summary>
+        /// <param name="endpoint">The endpoint to with the get request should be sent</param>
+        /// <param name="getParams"> A dictionary representing ke-value pairs that should be attached to the endpoint as GET parameters</param>
```

Comment:
Double check this file for typos, extra white space, and line length issues. (This line has all 3.)